.github/workflows/conformity-workflow.yml

```diff
@@ -0,0 +1,100 @@
+name: Conformity Test Workflow
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This should match the other workflows and should be more than just on main

Reply:
Changed it so it is as the other workflows